tests/backup/backup_helper_test.go

```diff
@@ -0,0 +1,1205 @@
+package tests
```

Comment:
do not mark as test file, name it as common

Reply:
done